pages/index.js

```diff
@@ -43,13 +56,12 @@ export default function Home() {
       <StyledList>
         <StyledHR />
```

Comment:
Same for the `hr` as mentioned above.

Reply:
Please see [here](https://github.com/MariaRiosNavarro/mobileSimulation-propOS/pull/15#discussion_r1238242569)

Reply:
All the suggestions have already been changed. And everything commented on by you. I have put the update of this PR on the channel, do I have to wait for someone else to correct it? Or through your correction it is not necessary and you put the green button on code reviews? I am confused on that point. Thank you